app/feature/feature-payments/src/main/kotlin/com/hedvig/android/feature/payments/discounts/DiscountsViewModel.kt

```diff
@@ -1,16 +1,16 @@
-package com.hedvig.android.feature.payments.overview
+package com.hedvig.android.feature.payments.discounts
 
 import com.hedvig.android.core.demomode.Provider
 import com.hedvig.android.feature.payments.overview.data.AddDiscountUseCase
 import com.hedvig.android.feature.payments.overview.data.GetPaymentOverviewDataUseCase
 import com.hedvig.android.molecule.android.MoleculeViewModel
 
-internal class PaymentOverviewViewModel(
+internal class DiscountsViewModel(
```

Comment:
The payments and the discounts screens were sharing the exact same ViewModel.
Here I just copy-pasted it to be used in the Discounts screen as-is, and later in the PR I made a new VM just for payments, since it was not using half of the stuff from in the VM and was making it hard for the UI to use this information, a lot of logic was delegated to the UI before.